test/integrationtests/voight_kampff/test_setup.py

```diff
@@ -0,0 +1,163 @@
+# limitations under the License.
+#
+import argparse
+from argparse import RawTextHelpFormatter
+from glob import glob
+from os.path import join, dirname, expanduser, basename
+from pathlib import Path
+from random import shuffle
+import shutil
+import sys
+
+import yaml
+
+from msm import MycroftSkillsManager
+from msm.exceptions import MsmException
+
+from .generate_feature import generate_feature
+
+"""Test environment setup for voigt kampff test
+
+The script sets up the selected tests in the feature directory so they can
+be found and executed by the behave framework.
+
+The script also ensures that the skills marked for testing are installed and
+that anyi specified extra skills also gets installed into the environment.
+"""
+
+FEATURE_DIR = join(dirname(__file__), 'features') + '/'
+
+
+def copy_feature_files(source, destination):
+    """Copy all feature files from source to destination."""
+    # Copy feature files to the feature directory
+    for f in glob(join(source, '*.feature')):
+        shutil.copyfile(f, join(destination, basename(f)))
+
+
+def copy_step_files(source, destination):
+    """Copy all python files from source to destination."""
+    # Copy feature files to the feature directory
+    for f in glob(join(source, '*.py')):
+        shutil.copyfile(f, join(destination, basename(f)))
+
+
+def load_config(config, args):
+    """Load config and add to unset arguments."""
+    with open(expanduser(config)) as f:
+        conf_dict = yaml.safe_load(f)
+
+    if not args.test_skills and 'test_skills' in conf_dict:
+        args.test_skills = conf_dict['test_skills']
+    if not args.extra_skills and 'extra_skills' in conf_dict:
+        args.extra_skills = conf_dict['extra_skills']
+    if not args.platform and 'platform' in conf_dict:
+        args.platform = conf_dict['platform']
+    return
+
+
+def main(cmdline_args):
+    """Parse arguments and run environment setup."""
+    platforms = list(MycroftSkillsManager.SKILL_GROUPS)
+    parser = argparse.ArgumentParser(formatter_class=RawTextHelpFormatter)
+    parser.add_argument('-p', '--platform', choices=platforms)
+    parser.add_argument('-t', '--test-skills', default=[],
+                        help=('Comma-separated list of skills to test.\n'
+                              'Ex: "mycroft-weather, mycroft-stock"'))
+    parser.add_argument('-s', '--extra-skills', default=[],
+                        help=('Comma-separated list of extra skills to '
+                              'install.\n'
+                              'Ex: "cocktails, laugh"'))
+    parser.add_argument('-r', '--random-skills', default=0,
+                        help='Number of random skills to install.')
+    parser.add_argument('-d', '--skills-dir')
+    parser.add_argument('-c', '--config',
+                        help='Path to test configuration file.')
+
+    args = parser.parse_args(cmdline_args)
+    if args.test_skills:
+        args.test_skills = args.test_skills.replace(',', ' ').split()
```

Comment:
don't need to call replace() and split() here.  you can just do `split(',')`

Reply:
I want to allow simple space separated list as well to a bit more lenient of semi-bad syntax.